### PR TITLE
Minor update to opening paragraph of csharp.md to be consistent.

### PR DESCRIPTION
**Pull request by gopher504**
Opening paragraph shall define what the style guide delivers. The open paragraph should be similar over all the style guide documents.

### CODE REVIEW

File: coding/csharp.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # C# Coding Standards
 
-The default language for back-end development is C#. The default IDE (Integrated Development Enviroment) for back-end development is Visual Studio. The exact requirements of the project will determine if any deviation form these defaults is required.
+This document defines formatting and style rules for C#. It aims at improving collaboration, code quality, and enabling supporting infrastructure. The default language for back-end development is C#. The default IDE (Integrated Development Enviroment) for back-end development is Visual Studio. The exact requirements of the project will determine if any deviation form these defaults is required.
```
Comment (petefitz) [inline coding/csharp.md]: one of its aims is also to "improve consistency across project streams"

Comment (gopher504) [inline coding/csharp.md]: Added.

**Comment by petefitz** [comment]
LGTM

**Comment by stevemchugh** [comment]
LGTM

**Comment by petefitz** [comment]
LGTM

**Pull request merged**